torch/nn/functional.py

```diff
@@ -565,12 +565,20 @@ def dropout3d(input, p=0.5, training=False, inplace=False):
     return _functions.dropout.FeatureDropout.apply(input, p, training, inplace)
 
 
-threshold = _add_docstr(torch._C._nn.threshold, r"""
-threshold(input, threshold, value, inplace=False) -> Variable
+def threshold(input, threshold, value, inplace=False):
+    """Thresholds each element of the input Tensor.
+
+    See :class:`~torch.nn.Threshold` for more details.
+    """
+    if inplace:
+        return torch._C._nn.threshold_(input, threshold, value)
```

Comment:
It's weird that now we're going to have two paths to apply these functions in-place. Maybe we should deprecate the arg?